### PR TITLE
Implement cephadm bootstrap

### PR DESCRIPTION
**Pull request by VasishtaShastry**
ceph octopus and corresponding rhcs release uses cephadm
for bootstrapping a cluster.
This commit has
 - New test to handle cephadm bootstrap.
 - New suite and conf file to test cephadm using test_cephadm module

Signed-off-by: VasishtaShastry <vipin.indiasmg@gmail.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: tests/misc_env/install_prereq.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def install_prereq(ceph, timeout=1800, skip_subscription=False, repo=False, rhbu
         # TODO enable only python3 rpms on both rhel7 &rhel8 once all component suites(rhcs3,4) are comptatible
         if distro_ver.startswith('8'):
             rpm_all_packages = ' '.join(rpm_packages.get('py3'))
+            if str(rhbuild).startswith('5'):
+                rpm_all_packages = rpm_all_packages + 'lvm2' + 'podman'
```
Comment (ktdreyer) [inline tests/misc_env/install_prereq.py:62]: If the `cephadm` package requires those two packages in order to function properly, but Yum does not automatically pull those in, that is a bug in my opinion.

**Comment by mykaul** [comment]
What's the latest status of this? This is critically important for us to run in CI using cephadm.
@veera-raghava-reddy  - anything we can help with this?

File: tests/misc_env/install_prereq.py (hunk 1 above)
Comment (VasishtaShastry) [reply tests/misc_env/install_prereq.py:62]: @ktdreyer Upstream doc says that  
> Dependencies are installed automatically by the bootstrap process below.
We will verify whether this behavior is happening and update the script accordingly and also file RFE if required.

Comment (ktdreyer) [reply tests/misc_env/install_prereq.py:62]: Thanks. I verified today that the latest cephadm RPM package has "Requires: lvm2" and "Recommends: podman". Maybe we should change that to "Requires: podman" now?

**Pull request closed without merging**